Show user and pack durations as minutes if less than an hour

alastair opened the pull request:
**Issue(s)**
Fixes https://github.com/MTG/freesound/issues/1818

**Description**
'0:03 hours' isn't very useful, better to show '3:45 minutes'
<img width="1289" alt="Screenshot 2025-02-21 at 9 23 15 PM" src="https://github.com/user-attachments/assets/e9235109-0582-4e4f-89dc-b02ebea63aeb" />

The code for computing/selecting hours vs minutes is duplicated in two places, for now I'm not worried about this but we could combine it in a helper if needed. The `|duration_hours` templatetag is still used elsewhere, so don't remove it.

I updated the test settings to disable the cache, so that I could read values for the test multiple times. I think it generally makes sense to disable the cache during tests? although some tests are now failing.
If we want to keep it then I can work around it for just this test.

ffont commented:
Maybe we should re-write that into a template filter and then have a test for the filter function only so we don't depend on the cache?

Cache is tested in different tests, I guess it makes sense to have it. But you should also be able to disable cache for a specific test using `@override_settings`? In any case I think it might be better to not test the template but some util function.

alastair commented:
Updated to use a templatetag that shows both the time and units, added a specific test for the templatetag and also general tests for the section templates.
I'm using some new pytest functionality for modifying settings, take a look!

ffont commented:
Thanks @alastair!
I noticed the `@pytest.mark.django_db` thingy. I guess this is useful to group tests for running them?
Also the fixture for disabling caches, which is passes as an argument in the test function.

I'll merge that one!
